docs/release-notes/cloud/cloud-2023-02.md

```diff
@@ -0,0 +1,83 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud February 2023.
+included_jira_versions: ['2023.02']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.76
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.7.116
+---
+
+# Cloud February 2023
+
+These release notes are for the Codacy Cloud updates during February 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   In some Enterprise plans, Codacy now automatically adds to the organization new people that commit to your private repositories and analyzes their commits, providing a more streamlined user management experience. (PLUTO-244)
```

Comment:
@AmhaBekele could you please check this release note in particular?

Reply:
LGTM